packages/reports/tests/integration/components/filter-values/lookback-input-test.js

```diff
@@ -60,4 +62,79 @@ module('Integration | Component | filter values/lookback input', function(hooks)
     assert.dom('.filter-values--lookback-input').doesNotExist('The lookback input does not display when collapsed');
     assert.dom().hasText(`4 days (${getDateRangeFormat(this)})`, 'The lookback is rendered correctly when collapsed');
   });
+
+  test('selecting preset values', async function(assert) {
+    assert.expect(4);
+
+    this.set('onUpdateFilter', filter => {
+      this.set('filter', { values: arr([filter.interval]) });
+    });
+
+    // Set to 7
+    await clickTrigger('.filter-values--lookback-input');
+    await click($('.navi-basic-dropdown-option:contains(7)')[0]);
+    assert.dom('.filter-values--lookback-input__value').hasValue('7', 'Clicking last 7 days changes input value to 7');
+
+    // Only 7 is selected
+    await clickTrigger('.filter-values--lookback-input');
+    assert.deepEqual(getSelectedOptions(), ['Last 7 Days'], '7 is the only selected option after being clicked');
+    await clickTrigger('.filter-values--lookback-input');
+
+    // Set to 30
+    await clickTrigger('.filter-values--lookback-input');
+    await click($('.navi-basic-dropdown-option:contains(30)')[0]);
+    assert
+      .dom('.filter-values--lookback-input__value')
+      .hasValue('30', 'Clicking last 30 days changes input value to 30');
+
+    // Only 30 is selected
+    await clickTrigger('.filter-values--lookback-input');
+    assert.deepEqual(getSelectedOptions(), ['Last 30 Days'], '30 is the only selected option after being clicked');
+    await clickTrigger('.filter-values--lookback-input');
+  });
+
+  test('typing in a predefined lookback value', async function(assert) {
+    assert.expect(2);
+
+    this.set('onUpdateFilter', filter => {
+      this.set('filter', { values: arr([filter.interval]) });
+    });
+
+    // type in 14
+    await fillIn('.filter-values--lookback-input__value', 14);
+    await blur('.filter-values--lookback-input__value');
+    assert.dom('.filter-values--lookback-input__value').hasValue('14', 'typing in 14 updates the value');
+
+    // Only 14 is selected
+    await clickTrigger('.filter-values--lookback-input');
+    assert.deepEqual(
+      getSelectedOptions(),
+      ['Last 14 Days'],
+      '14 is the only selected in the dropdown after being typed in'
+    );
+    await clickTrigger('.filter-values--lookback-input');
+  });
+
+  test('typing in a not predefined lookback value', async function(assert) {
```

Comment:
What happens if you first select a predefined value then change it to a non-predefined values either through clicking the arrows or manually typing it in?

Reply:
Still works, I'll add it in the test though